workspace/notebook/py_unit_tests_functions.json

```diff
@@ -480,6 +480,8 @@
 					"                                    FROM {curated_db}.{curated_table}\r\n",
 					"                                )\r\n",
 					"                            \"\"\")\r\n",
+					"    if df.count() > 0:\r\n",
```

Comment:
could this be removed please? Think we only needed this for debugging

Reply:
true, but there are other places we leave stuff in for debugging such as 

```
standardised_count, harmonised_count, counts_match = test_std_same_rows_hrm(std_table_name, hrm_table_name)
print(f"Standardised Count: {standardised_count: ,}\nHarmonised Count: {harmonised_count: ,}\nCounts match: {counts_match}")

if standardised_count > harmonised_count:
    exitCode += 1
    print(f"{standardised_count - harmonised_count} rows from Standardised are missing in Harmonised." )
    differentiate_std_and_hrm(f"{std_db_name}.{std_table_name}", f"{hrm_db_name}.{hrm_table_name}", data_model_columns)
```
    
happy to take it out, but I think it's useful enough to keep in

Reply:
yeah, probably we can discuss this in a separate issue whether it's best to keep/comment/remove debugging in our notebooks